index.html

```diff
@@ -19,8 +19,168 @@ <h1>Sebastián<br>Muñoz</h1>
     <section class="intro">
         <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </section>
-    <div class="collab">
+    <div class="btn">
         <button type="button">Start collaboration</button>
     </div>
+    <div class="geometry">
+        <img src="assets/img/image_geometry_2.png" alt="">
+    </div>
+    <section class="projects-title">
+        <h1>Projects</h1>
+        <ul class="projects">
+            <li>
+                <div class="grid-container">
+                    <div class="item1">
+                        <img src="assets/img/Project1.png" alt="">
+                    </div>
+                    <div class="item2">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <div class="item3">HTML/CSS</div>  
+                    <div class="item4">Ruby on Rails</div>
+                    <div class="item5">JavaScript</div>
+                    <div class="item6"></div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are supposed to use list `ul` and each of them a `li`. It will help group the list semantically.